### PR TITLE
[Fix](Full compaction) Fix full compaction by table id case

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

In previous tests, all tablets were not waited to finish full compaction, which resulted in the failure of the CI. This PR fixes this issue.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

File: regression-test/suites/compaction/test_full_compaction_by_table_id.groovy [hunk 1]
```diff
@@ -139,18 +139,20 @@ suite("test_full_compaction_by_table_id") {
...
-                def compactionStatus = parseJson(out.trim())
-                assertEquals("success", compactionStatus.status.toLowerCase())
-                running = compactionStatus.run_status
-            } while (running)
+            for (int i=0; i<tablets.size(); ++i) {
+                boolean running = true
+                do {
+                    Thread.sleep(1000)
+                    tablet_id = tablets[i][0]
+                    backend_id = tablets[0][2]
```
Comment (zhannngchen) [inline regression-test/suites/compaction/test_full_compaction_by_table_id.groovy:147]: should be `tablets[i][2]`?

**Comment by zhannngchen** [comment]
you should also consider to trigger the compaction on every BE

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Pull request merged**